src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/visits/VisitsMigrationService.kt

```diff
@@ -183,6 +229,9 @@ class VisitsMigrationService(
   }
 }
 
+private fun <T> MigrationContext<T>.durationMinutes(): Long =
+  Duration.between(LocalDateTime.parse(this.migrationId), LocalDateTime.now()).toMinutes()
+
```

Comment:
for now relay on the migrationId being the date time migration started - if we ever made the migrationId a GUUID then obviously this would have to change